system_baseline/internal_views/v1.py

```diff
@@ -9,7 +9,7 @@
 
 section = Blueprint("internal_v1", __name__)
 
-FACTS_MAXSIZE = 2**19  # 512KB
+FACTS_MAXSIZE = 2 ** 19  # 512KB
```

Comment:
See my comment above (about irrelevant changes). Fixup any formatting changes to commits that have introduced those issue(s).